FiligranHQ/filigran-ui
# [filigran-ui] : (frontend) add combobox in ui and website

This PR adds the Combox into the DS. 

## how to test 
- Connect on the Filigran Website
- Go to "Docs" Then search "Combobox" in the components. 
- Check the combobox works well. 

Test OK on local 🆗 :
![image](https://github.com/FiligranHQ/filigran-ui/assets/169655135/88be19c6-e65d-4306-9190-31d247916585)

![image](https://github.com/FiligranHQ/filigran-ui/assets/169655135/ea446c68-a48f-4e5f-9b75-ca490998d46f)

![image](https://github.com/FiligranHQ/filigran-ui/assets/169655135/7a76c9a0-4a62-40a7-bd8e-266941f46f72)
